tests/stdlib/tstrformat.nim

```diff
@@ -6,7 +6,7 @@ output: '''Received (name: "Foo", species: "Bar")'''
 # issue #7632
 
 import genericstrformat
-import strutils, times
+import std/[strutils, times]
```

Comment:
pre-existing but since you're cleaning up the tests:
* remove `action: "run"` (redundant)
* see if you can use `doAssert` instead of `output: '''Received (name: "Foo", species: "Bar")'''`

Reply:
I disabled JS and VM tests for now. A later PR should fix that.